### ISSUE TITLE
[🐛 | Bug]: Invalid entrypoint in parsec-server docker image

### ISSUE THREAD

**Issue by FirelightFlagboy** [Scille/parsec-cloud#5365]
### Version

2.16

### What platforms you were running our software when the bug happened ?

Linux

### What happened ?

Currently the docker container use the following entrypoint:

```shell
/backend/venv/bin/python parsec backend
```

This cause error because `python` will try to look for a file named `parsec` in the current working directory, file that doesn't exist and result in the following error:

```
/backend/venv/bin/python: can't open file '/backend/parsec': [Errno 2] No such file or directory
```

#### Workaround

Until the issue is fixed we can overwrite the `entrypoint` to:

```shell
/backend/venv/bin/python /backend/venv/bin/parsec backend
```

#### In 2.16

https://github.com/Scille/parsec-cloud/blob/9cdaec0075dea51108c648dc86dc843a4163a5fa/packaging/server/server.dockerfile#L43

#### In master

https://github.com/Scille/parsec-cloud/blob/be7eef1a11fedb4bc15cc312823abe536c416cb6/server/packaging/server/server.dockerfile#L43

**Comment by chinmaym07** [comment]
Hey @FirelightFlagboy 
Can I contribute to this ?

**Comment by FirelightFlagboy** [comment]
Hello @chinmaym07,

Sure, if you're interested to contribute to this issue and open a pull-request.

If not done already, I suggest you to read how to contribute to this repo at:

https://github.com/Scille/parsec-cloud/blob/4c2c08c92eacc71ecb6a2c0b6a4cc472d9426745/CONTRIBUTING.md?plain=1#L44-L55

**Comment by FirelightFlagboy** [comment]
Resolved in #5437
